sql/driver/driver_test.go

```diff
@@ -31,7 +32,7 @@ var isError = testutils.IsError
 
 func setup(t *testing.T) (*server.TestServer, *sql.DB) {
 	s := server.StartTestServer(nil)
-	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	db, err := sql.Open("cockroach", "https://"+security.RootUser+"@"+s.ServingAddr()+"?certs=test_certs")
```

Comment:
We're unlikely to change `security.RootUser` to have any other value, but we do want to discourage use of hard-coded usernames and especially `root`. We should really have StartTestServer() create a non-root user with a non-hard-coded name and use that everywhere instead of hard-coding `RootUser`.